modules/installation-restricted-network-samples.adoc

```diff
@@ -30,7 +30,7 @@ not addressed in this procedure.
 in the restricted network environment into one of the defined mirrors:
 +
 ----
-$ oc image mirror myregistry.com/myimage:latest myregistry.com/myimage:stable
+$ oc image mirror registry.redhat.io/openshift4/ose-jenkins-agent-maven:latest myregistry.com/ose-jenkins-agent-maven:latest
```

Comment:
There is no second slash in `myregistry.com/ose-jenkins-agent-maven:latest`

Reply:
ironically jenkins is one of the ones that does not need to be explicitly mirrored because the samples imagestream for jenkins imports from the payload and uses SHA.  So please pick a different example.

Reply:
Correct ... simply changing the `samplesRegistry` setting on the samples operator config to the mirror registry should be sufficient, assuming you keep the repository and image names (minus sha/tag/etc) the same

Reply:
> simply changing the samplesRegistry setting on the samples operator config to the mirror registry should be sufficient, assuming you keep the repository and image names (minus sha/tag/etc) the same

You still need to mirror the content which is what this is trying to illustrate.  It's just that the jenkins image in particular doesn't need to be mirrored because it should have already been mirrored as part of the disconnected install process.

Reply:
sorry yes I was assuming you mirrored previously .. .after that, configure the `samplesRegistry`